modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgniteDataStorageMetricsSelfTest.java

```diff
@@ -389,15 +385,78 @@ public void testWalWrittenBytes() throws Exception {
         assertEquals(exp, ((LongAdderMetric)dsMetricRegistry(n).findMetric("WalWrittenBytes")).value());
     }
 
+    /**
+     * Check whether WAL is reporting correct usage when archiving was not needed.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testWalTotalSizeWithoutArchivingPerfomed() throws Exception {
+        IgniteEx n = startGrid(0,
+                     (Consumer<IgniteConfiguration>)cfg -> cfg.getDataStorageConfiguration()
+                     .setWalSegmentSize((int)(2 * U.MB)));
+
+        n.cluster().state(ACTIVE);
+
+        populateCache(n);
+
+        turnOffWal(n);
+
+        checkWalArchiveAndTotalSize(n, true);
+    }
+
+    /**
+     * Check whether WAL is reporting correct usage when archiving is performed.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testWalTotalSizeWithArchive() throws Exception {
+        IgniteEx n = startGrid(0,
```

Comment:
Wrong style, either all on one line or on every line.
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-MethodParameters

Reply:
Not fixed.